README.md

```diff
@@ -151,42 +211,79 @@ The functional_test makefile target is used for running end-to-end tests:
+Please have a look to the section
+[on building the source code](doc/manuals/admin/build_source.md) in order to get
+more information about how to prepare the environment to run the unit_test
+target.
 
 [Top](#top)
 
 ## Advanced topics
 
-* Advanced Programming [(en)](doc/manuals/user/README.md) [(jp)](doc/manuals.jp/user/README.md)
-* Installation and administration [(en)](doc/manuals/admin/README.md) [(jp)](doc/manuals.jp/admin/README.md)
-* Container-based deployment
-  * Docker [(en)](docker/README.md) [(jp)](docker/README.jp.md)
-  * Docker Swarm and HA [(en)](docker/docker_swarm.md) [(jp)](docker/docker_swarm.jp.md)
-* Development Manual [(en)](doc/manuals/devel/README.md) [(jp)](doc/manuals.jp/devel/README.md)
-* Sample code contributions [(en)](doc/manuals/code_contributions.md) [(jp)](doc/manuals.jp/code_contributions.md)
-* Contribution guidelines [(en)](doc/manuals/contribution_guidelines.md) [(jp)](doc/manuals.jp/contribution_guidelines.md), especially important if you plan to contribute with code
-  to Orion Context Broker
-* Deprecated features [(en)](doc/manuals/deprecated.md) [(jp)](doc/manuals.jp/deprecated.md)
+-   Advanced Programming [(en)](doc/manuals/user/README.md)
+    [(jp)](doc/manuals.jp/user/README.md)
+-   Installation and administration [(en)](doc/manuals/admin/README.md)
+    [(jp)](doc/manuals.jp/admin/README.md)
+-   Container-based deployment
+    -   Docker [(en)](docker/README.md) [(jp)](docker/README.jp.md)
+    -   Docker Swarm and HA [(en)](docker/docker_swarm.md)
+        [(jp)](docker/docker_swarm.jp.md)
+-   Development Manual [(en)](doc/manuals/devel/README.md)
+    [(jp)](doc/manuals.jp/devel/README.md)
+-   Sample code contributions [(en)](doc/manuals/code_contributions.md)
+    [(jp)](doc/manuals.jp/code_contributions.md)
+-   Contribution guidelines [(en)](doc/manuals/contribution_guidelines.md)
+    [(jp)](doc/manuals.jp/contribution_guidelines.md), especially important if
+    you plan to contribute with code to Orion Context Broker
+-   Deprecated features [(en)](doc/manuals/deprecated.md)
+    [(jp)](doc/manuals.jp/deprecated.md)
 
 [Top](#top)
 
-## License
+## Support
 
-Orion Context Broker is licensed under Affero General Public License (GPL) version 3.
+Ask your thorough programming questions using
+[stackoverflow](http://stackoverflow.com/questions/ask) and your general
+questions on [FIWARE Q&A](https://ask.fiware.org). In both cases please use the
+tag `fiware-orion`
 
 [Top](#top)
 
-## Support
+## Quality Assurance
 
-Ask your thorough programming questions using [stackoverflow](http://stackoverflow.com/questions/ask)
-and your general questions on [FIWARE Q&A](https://ask.fiware.org). In both cases please use the tag `fiware-orion`
+This project is part of [FIWARE](https://fiware.org/) and has been rated as
+follows:
 
-[Top](#top)
+-   **Version Tested:**
```

Comment:
Same comment to the one published at https://github.com/telefonicaid/iotagent-ul/pull/301#discussion_r236292514